sdk/tables/aztable/byte_array_response.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+// ByteArrayResponse converts the MapOfInterfaceResponse.Value from a map[string]interface{} to a []byte
+type ByteArrayResponse struct {
+	// ClientRequestID contains the information returned from the x-ms-client-request-id header response.
+	ClientRequestID *string
+
+	// ContentType contains the information returned from the Content-Type header response.
+	ContentType *string
+
+	// Date contains the information returned from the Date header response.
+	Date *time.Time
+
+	// ETag contains the information returned from the ETag header response.
+	ETag *string
```

Comment:
for future - in some of the other languages ETag is a type in Core. I wonder if we should do that eventually here.

Reply:
Yes, I agree. azcore should have an etag type. @jhendrixMSFT

Reply:
https://github.com/Azure/azure-sdk-for-go/issues/15153